natlas-server/webpack.config.js

```diff
@@ -57,8 +46,8 @@ const config = (env, argv) => {
                 manifestPath: path.resolve(assetRootPath, 'dist', 'webpack_manifest.json'),
                 outputRoot: assetRootPath
             }),
-            new webpack.DefinePlugin({
-                NATLAS_VERSION: JSON.stringify(revision())
+            new webpack.EnvironmentPlugin({
+                NATLAS_VERSION: "dev"
```

Comment:
Man this is not intuitive at all that this is the "default" value only if `NATLAS_VERSION` isn't in the environment.